src/helper/plasmUtils.ts

```diff
@@ -26,6 +26,12 @@ export const reduceDenomToBalance = (
   return formatted;
 };
 
+export const convertToBalance = (bal: BN, unit: number) => {
```

Comment:
You can use existing helper functions like this https://github.com/polkadot-js/common/blob/master/packages/util/src/format/formatDecimal.ts and https://github.com/polkadot-js/common/blob/master/packages/util/src/format/formatBalance.ts which is more accurate.